test/integration.js

```diff
@@ -415,14 +415,19 @@ tape('should close channel', async function(t) {
 	await postEvsAsCreator(leaderUrl, channel.id, genEvents(10))
 
 	// close channel event
+	// Do not use genEvents because it adds default publisher which add event payout
```

Comment:
No, it doesn't because it's a `CLOSE` event it won't add event payout for the default publisher. Since we already verify it's the creator the remaining balance is awarded automatically to the creator address. 

Also, can we revert the event creation to the previous implementation? With the previous implementation, we award the remaining balance to the creator regardless of the publisher. 

If a regression/bug were introduced in the future that awards it to the publisher input the new test won't enable us to catch it.

Reply:
if `genEvents` 
https://github.com/AdExNetwork/adex-validator/blob/bebf3e295af56022c280d118bce44eccc7af4dc1/test/lib.js#L43

if `[{type: 'CLOSE', publisher: defaultPubName }]`
https://github.com/AdExNetwork/adex-validator/blob/bebf3e295af56022c280d118bce44eccc7af4dc1/services/sentry/eventAggregator.js#L88
https://github.com/AdExNetwork/adex-validator/blob/bebf3e295af56022c280d118bce44eccc7af4dc1/services/sentry/lib/getPayout.js#L12

then goes here which is not correct ad it adds the publisher as earner which is not the case with "CLOSE" event
https://github.com/AdExNetwork/adex-validator/blob/bebf3e295af56022c280d118bce44eccc7af4dc1/services/sentry/lib/eventReducer.js#L10
we need it only here:
https://github.com/AdExNetwork/adex-validator/blob/bebf3e295af56022c280d118bce44eccc7af4dc1/services/sentry/lib/eventReducer.js#L20

when the test had event with publisher it was passing before the fix of `isEmpty` because there was earners which is not correct

Reply:
Yeah, the issue is changing it doesn't fix the `earners` array problem. We can still add a publisher to the `CLOSE` event and it will be added to the `earners` array. 

We should add an additional check `getPayout` to remove future possibilities
this line should become
https://github.com/AdExNetwork/adex-validator/blob/master/services/sentry/lib/getPayout.js#L9

```	
if (!ev.publisher || ev.type === eventTypes.close) return null
```

Reply:
Yep, you are right but instead `ev.type === eventTypes.close` I think is better to check the type is `CLICK` or 'IMPRESSION' only as there it 'UPDTE_TARGETING' event and maybe others in the future